### PR TITLE
Fix AppSegments for Reader

### PR DESCRIPTION
**Pull request by nickheiner-usds**
The loading spinner and error message were erroneously not being placed in `AppSegment`s. This fixes that.

### CODE REVIEW

File: client/app/reader/CaseSelect.jsx [hunk 1]
```diff
@@ -71,8 +71,7 @@ class CaseSelect extends React.PureComponent {
       return null;
     }
 
-    return <AppFrame wideApp>
```
Comment (nickheiner-usds) [inline client/app/reader/CaseSelect.jsx]: We're pulling the `AppFrame` up a level so we don't have to duplicate it for the loading spinner.

File: client/app/reader/DecisionReviewer.jsx [hunk 2]
```diff
@@ -154,17 +155,19 @@ export class DecisionReviewer extends React.PureComponent {
...
-            path="/"
-            title="Assignments | Caseflow Reader"
-            render={this.routedCaseSelect} />
-          <PageRoute
-            exact
-            title={this.getClaimsFolderPageTitle(this.props.appeal)}
-            breadcrumb="Claims Folder"
-            path="/:vacolsId/documents"
-            render={this.routedPdfListView} />
+          <AppFrame wideApp>
```
Comment (mdbenjam) [inline client/app/reader/DecisionReviewer.jsx]: Something I just noticed. Can you move these pageroutes above the document viewer one? Currently the bread crumbs are out of order when you're looking at the pdf viewer.

Comment (nickheiner-usds) [reply client/app/reader/DecisionReviewer.jsx]: Good catch. I'll just fix it by sorting the breadcrumbs on `path.length` so the mechanism is less error-prone generally.

Comment (nickheiner-usds) [reply client/app/reader/DecisionReviewer.jsx]: https://github.com/department-of-veterans-affairs/caseflow-frontend-toolkit/pull/12

**Pull request merged**